deeplab/src/config.ts

```diff
@@ -0,0 +1,384 @@
+export const config = {
+  BASE_PATH: 'https://storage.googleapis.com/gsoc-tfjs/models/deeplab',
```

Comment:
move to an official path away from gsoc-tfjs may be?

Reply:
Yes, that is something that Core team will do before publishing it. :)

Reply:
@startupgurukul, there is also a [script](https://github.com/sdll/tfjs-models/blob/models/deeplab/deeplab/scripts/convert_deeplab.sh) to convert the weights.